### PR TITLE
Add pluck and assign

### PR DESCRIPTION
**Pull request by svozza**

### CODE REVIEW

**Comment by cullophid** [comment]
Does assign deepclone all the given objects ?

**Comment by svozza** [comment]
No, it's a shallow clone. For this,  I wasnt sure how far we wanted to differ from the ES6 `assign` function.

**Comment by cullophid** [comment]
I think ist fine as long as we are not modifying them.

**Comment by svozza** [comment]
Yeah, me too. I'll put a note in the docs though.

File: README.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ _.append(5, [1, 2, 3, 4]);
 // => [1, 2, 3, 4, 5]
 ```
 
+### assign :: [Object] -> Object
```
Comment (cullophid) [inline README.md]: Shouldnt we call it merge all? Since its no longer assign?

Comment (cullophid) [inline README.md]: And maybe add merge aswell

Comment (svozza) [inline README.md]: Yeah, I wasn't sure about that but I definitely prefer `mergeAll` as a name. Do we need `merge` though if you can just pass in an array of two objects to get the same functionality?

Comment (cullophid) [inline README.md]: Yeah but merge is curried. I use it quite a a lot. 

```js
const prepareOptions = _.merge(DEFAULTS)
```

Comment (svozza) [inline README.md]: True, I'll as it then.

**Comment by svozza** [comment]
Is this good to go?

**Pull request merged**